FrameworkSystem/private/standardLogging/Handler/RemoteHandler.py

```diff
@@ -0,0 +1,94 @@
+"""
```

Comment:
It is almost exactly the same as the old RemoteBackend. This class send log messages to the DIRAC service "SystemLogging". 
I changed comments, do you understand now ?